parser/configs/env.go

```diff
@@ -0,0 +1,17 @@
+package configs
+
+import (
+	"log"
+	"os"
+
+	"github.com/joho/godotenv"
+)
+
+func EnvMongoURI() string {
+	err := godotenv.Load()
```

Comment:
Is it possible to import this from the api, or include this in the parser.go? Doing this will minimize confusion within code speculators in IDEs when autocompleting function names.

Reply:
We are going to be seperating the API from our tooling, so this is fine.